File: eighth/python/2.py
def converter_kelvin_para_celsius_fahrenheit(temp_kelvin):
    temp_celsius = temp_kelvin - 273.15
    temp_fahrenheit = (temp_kelvin - 273.15) * 9/5 + 32
    return temp_celsius, temp_fahrenheit

# Exemplo de uso
temp_kelvin = 300
temp_celsius, temp_fahrenheit = converter_kelvin_para_celsius_fahrenheit(temp_kelvin)
print(f"A temperatura em graus Celsius é: {temp_celsius:.2f}°C")
print(f"A temperatura em graus Fahrenheit é: {temp_fahrenheit:.2f}°F")

File: eighth/c/2.c
#include <stdio.h>

void converterKelvinParaCelsiusFahrenheit(float tempKelvin, float *tempCelsius, float *tempFahrenheit) {
    *tempCelsius = tempKelvin - 273.15;
    *tempFahrenheit = (tempKelvin - 273.15) * 9/5 + 32;
}

int main() {
    float tempKelvin = 300;
    float tempCelsius, tempFahrenheit;
    converterKelvinParaCelsiusFahrenheit(tempKelvin, &tempCelsius, &tempFahrenheit);
    printf("A temperatura em graus Celsius é: %.2f°C\n", tempCelsius);
    printf("A temperatura em graus Fahrenheit é: %.2f°F\n", tempFahrenheit);
    return 0;
}
